q2_vsearch/plugin_setup.py

```diff
@@ -430,7 +430,8 @@
                  'identify chimeric feature sequences. '
                  'The results of these methods can be used to filter chimeric '
                  'features from the corresponding feature table. '
-                 'For more details, please refer to the vsearch manual.')
+                 'For more details, please refer to the vsearch manual.'),
+    citations=[citations['edgar2011uchime', 'edgar2016uchime2', 'edgar2016unoise2']]
```

Comment:
@colinvwood Is there a good way to list only one of these depending on method? Is it fine to include all of them?

Reply:
There's no way to use only one depending on the algorithm without splitting each algorithm into a new action. But this is fine I think.